### ISSUE TITLE
blind accessibility issues

### ISSUE THREAD

**Issue by iampremo** [YoYoGames/GameMaker-Bugs#3727]
[Ticket #187094]

 

Feature Type: feat_addition_ingame
 Description: I am wondering if it can be done to make the product in general accessible with screen raeders such as jaws NVDA voiceover and others. I cannot operate game maker 2 and beraly have any access with gm 1.4. This is very dificult since there is a couple of games made for blind such as the abandoned project for porting pokemon to a blind audio game format.
 Benefit: This would allow even low and or blind users to even make blind audio games.
 Link: There is info on the screen readers that is used such as NVDA and jaws. jaws is the screen reader from Freedom Scientific and NVDA is a free screen reader, as for Voiceover that is built in all Apple products such as IOS ipad IOS apple watch and macs.

[Ticket #193479] - Further request: 

Feature Type: feat_addition_ide
Description: Hello!

This is a simple request, right now Windows Narrator and similar programs such as NV Access can not see anything inside of the IDE. This makes the use of GMS2 simply impossible for the visually impaired and the fully blind. Since GMS2 is branded as a game engine for everyone, this is disappointing.
Benefit: Simply, I can tell fans that they can make games as well. I make Audio games that are blind accessible for people who are fully blind. But at the moment there are 0 main stream game engines that are even some what blind accessible.

Since GMS2 is primarily 2D, but has 3D sound capabilities, it is actually perfect for making audio games. But people who are fully blind who want to use it can't since it's not screen reader friendly. If it was made so that screen readers could work on it, I'm confident that blind and visually impaired developers would be able to make games easier than ever.

Currently every developer that is blind that I speak too either uses incredibly old engines that have been modded, or make their own simplified game engines from scratch.

Here are my personal audio games I made using GMS2 for game jams in last several months:
[https://ericbomb.itch.io/]
Here are the ones that scored incredibly well in standard game jams:
This one got [#2|Ticket #2] in audio in Brackeys and [#10|Ticket YoYoGames/GameMaker-Feature-Requests#10] in innovation (Overall score was lower as I was near the bottom in graphics score)
[https://itch.io/jam/brackeys-6/rate/1172596]

This one got [#1|Ticket #1] in concept and [#8|Ticket YoYoGames/GameMaker-Feature-Requests#8] overall in a Major Jam on itch.io
[https://itch.io/jam/major-jam-4-cosmic/rate/1160835]

So the potential for GMS2 to be used to make audio games feels very large, but right now the people who would most want to make audio games can't use it. So just hoping that screen reader accessibility would be added. The ability to inspire blind developers and be able to help them learn GMS2 would simply mean the world to me.

**Comment by YYDan** [comment]
## Duplicate old request: 

### Description

Hello! This is a simple request, right now Windows Narrator and similar programs such as NV Access can not see anything inside of the IDE. This makes the use of GMS2 simply impossible for the visually impaired and the fully blind. Since GMS2 is branded as a game engine for everyone, this is disappointing.

### Benefit

Simply, I can tell fans that they can make games as well. I make Audio games that are blind accessible for people who are fully blind. But at the moment there are 0 main stream game engines that are even some what blind accessible.

Since GMS2 is primarily 2D, but has 3D sound capabilities, it is actually perfect for making audio games. But people who are fully blind who want to use it can't since it's not screen reader friendly. If it was made so that screen readers could work on it, I'm confident that blind and visually impaired developers would be able to make games easier than ever. Currently every developer that is blind that I speak too either uses incredibly old engines that have been modded, or make their own simplified game engines from scratch. So the potential for GMS2 to be used to make audio games feels very large, but right now the people who would most want to make audio games can't use it. So just hoping that screen reader accessibility would be added. The ability to inspire blind developers and be able to help them learn GMS2 would simply mean the world to me.
